SFBaseKit/Utils/Observable.swift

```diff
@@ -32,12 +32,49 @@ public final class Observable<T> {
     }
     
     /// Attaches a closure to execute when the binding value changes.
+    /// The closure is executed on attachment.
+    /// - Parameter receiveValue: The closure to execute with the new value.
+    public func sinkAndFire(on dispatchQueue: DispatchQueue = .main, receiveValue: @escaping (T) -> Void) {
+        subject
+            .receive(on: dispatchQueue)
+            .sink(receiveValue: receiveValue)
+            .store(in: &cancellables)
+    }
+    
+    /// Attaches a closure to execute when the binding value changes.
+    /// The closure is not executed on attachment.
     /// - Parameter receiveValue: The closure to execute with the new value.
     public func sink(on dispatchQueue: DispatchQueue = .main, receiveValue: @escaping (T) -> Void) {
         subject
+            .dropFirst()
             .receive(on: dispatchQueue)
             .sink(receiveValue: receiveValue)
             .store(in: &cancellables)
     }
+    
+    /// Binds the observable value to the values received from a Bindable's Publisher.
+    /// - Parameters:
+    ///   - bindable: A bindable object to receive values from.
```

Comment:
I suggest we stick to one pattern when listing parameters.
_/// Documentation option one
/// - Parameters:
///   - firstParameterName:
///   - secondParameterName:_

_/// Documentation option two
/// - Parameter firstParameterName:
/// - Parameter secondParameterName:_ 

The two different options are used in the _Observable_ and _Bindable_ files. I also suggest to we document all the parameters.

Reply:
Updated parameter documentation